bin/development-server.js

```diff
@@ -1,6 +1,8 @@
 #!/usr/bin/env node
 "use strict";
 
+require("babel-register");
```

Comment:
now that feature.js references `devtools-sham` we need babel-register to do the lookup. This hopefully will go away soon though